### DISCUSSION TITLE
[A/L] (2021/22) Logica - Esercizio 7

### DISCUSSION THREAD

**Discussion by matypist** [sapienzastudentsnetwork/metodi-matematici-per-l-informatica#333] [Esercizi A-L]
![image](https://user-images.githubusercontent.com/69389461/152324558-2c8c993a-c7be-4e80-9422-ab5f601fa033.png)

**Comment by danieldinella** [comment]
A → B = 1 solo se B = 1 o A = 0 e B = 0.
Consideriamo A  →  B = 1.
1 == A ⟷ (A^1) ovvero 1 == A ⟷ A dunque 1==1.
Consideriamo A  →  B = 0.
0 == 1 ⟷ (1^0) ovvero 0 == 1 ⟷ 0 dunque 0==0.
La proposizione è sempre dimostrata

A → B = 1 solo se B = 1 o A = 0 e B = 0.
Consideriamo A  →  B = 1.
1 == 1 ⟷ (A v 1) ovvero 1 == 1 ⟷ 1 dunque 1==1.
Consideriamo A  →  B = 0.
0 == 0 ⟷ (1 v 0) ovvero 0==0 ⟷ 1 dunque 0==0.
La proposizione è sempre dimostrata

A ⟷ B=0 solo se A!=B.
Consideriamo A ⟷ B=0.
L'and tra due termini diversi da 0 mentre l'or da 1, dunque 0 == 0 ⟷ 1 ovvero 0==0.
Consideriamo A ⟷ B=1.
Se A = B = 0, abbiamo 0 == 1  ⟷  0 dunque 0 == 0.
Se A = B = 1 abbiamo 1 == 1 ⟷ 1 dunque 1 == 1.

A ⟷ B=0 solo se A!=B.
Consideriamo A ⟷ B=0.
L'and tra due termini diversi da 0 mentre l'or da 1, dunque 0 == 1 → 0 ovvero 0==0.
Consideriamo A ⟷ B=1.
Se A = B = 0, abbiamo 1 == 0  →  0 dunque 1 == 1.
Se A = B = 1 abbiamo 1 == 1  →  1 dunque 1 == 1.

**Comment by Exyss** [comment]
Condivido delle soluzioni alternative a quelle di @danieldinella, ottenute utilizzando gli assiomi dell'algebra di Boole
e delle equivalenze notevoli

![Esercizio 7](https://user-images.githubusercontent.com/65043207/152681265-9a30436d-fd4e-48e8-82f6-4c01b8985f1f.jpg)

**Comment by CiottoloMaggico** [comment]
![immagine](https://user-images.githubusercontent.com/79638284/217805898-c6e17fc5-a2ba-474d-8e1d-6d95875775d8.png)
